app/views/orga/teams/show.html.slim

```diff
@@ -74,12 +74,11 @@ nav.btn-group
   h3 Supervisors
   p = @team.supervisors.any? ? link_to_team_members(@team, :supervisor) : content_tag(:em, 'Will be added by RGSoC Organizers')
 
-- if @team.members.any? { |user| user.conferences.in_current_season.any? }
+- if @team.conferences.in_current_season.any?
   h4 Conferences
   ul.conferences
-    - @team.members.each do |user|
-      - if user.conferences.in_current_season.any?
-        li = "#{link_to user.name, user}: #{links_to_conferences(user.conferences.in_current_season).join(', ')}".html_safe
+      - if @team.conferences.in_current_season.any?
+        li = "#{link_to @team.name, @team}: #{links_to_conferences(@team.conferences.in_current_season).join(', ')}".html_safe
```

Comment:
I did not look at this in the browser, maybe I'm getting it wrong. But in my understanding this should be a list of conferences. So something like:
```ruby 
ul
  li conference_1
  li conference_2
  # etc.
```

The ways your view code looks to me, this will only render a single `li` item (will all conferences in it...).